dev/NavigationView/NavigationView.idl

```diff
@@ -363,7 +363,7 @@ unsealed runtimeclass NavigationViewItem : NavigationViewItemBase
         static Windows.UI.Xaml.DependencyProperty MenuItemsSourceProperty{ get; };
     }
 
-    [MUX_PREVIEW]
+    [MUX_PUBLIC]
```

Comment:
This and presenter needs to be V2

Reply:
Why that?

Reply:
Because it is a new property that is being added to an existing type. This helps keep the the binary be ABI compatible.

Reply:
Got it, thanks!